packages/semantic-model/src/api.ts

```diff
@@ -16,15 +16,22 @@ export function generate({
   version,
   libraries,
   typeNameFix,
-  strict = true
+  strict = true,
+  printValidationErrors = true
 }: {
   version: string;
   libraries: Record<string, Json>;
   typeNameFix: TypeNameFix;
   strict?: boolean;
+  printValidationErrors?: boolean;
 }): UI5SemanticModel {
   const jsonSymbols = newMap<ConcreteSymbol>();
-  const model = convertToSemanticModel(libraries, jsonSymbols, strict);
+  const model = convertToSemanticModel(
```

Comment:
if/when we need to add another param (in future) to `convertToSemanticModel` we should probably use "config object pattern"